temporalio/client.py

```diff
@@ -3851,6 +4156,39 @@ class TerminateWorkflowInput:
     rpc_timeout: Optional[timedelta]
 
 
+@dataclass
+class UpdateWorkflowInput:
+    """Input for :py:meth:`OutboundInterceptor.start_workflow_update`."""
+
+    id: str
+    run_id: Optional[str]
+    update_id: Optional[str]
+    update: str
+    args: Sequence[Any]
+    wait_for_stage: Optional[
+        temporalio.api.enums.v1.UpdateWorkflowExecutionLifecycleStage.ValueType
+    ]
+    headers: Mapping[str, temporalio.api.common.v1.Payload]
+    ret_type: Optional[Type]
+    rpc_metadata: Mapping[str, str]
+    rpc_timeout: Optional[timedelta]
+
+
+@dataclass
+class PollUpdateWorkflowInput:
+    """Input for :py:meth:`OutboundInterceptor.poll_workflow_update`."""
+
+    workflow_id: str
+    run_id: Optional[str]
+    update_id: str
+    update: str
+    timeout: Optional[timedelta]
+    headers: Mapping[str, temporalio.api.common.v1.Payload]
```

Comment:
I don't think there's a concept of headers for this call (so no tracing on it either)

Reply:
Ah, yeah I suppose that makes sense